--- kaddressbook-26.04.3/CMakeLists.txt~	2026-06-28 17:25:15.000000000 +0000
+++ kaddressbook-26.04.3/CMakeLists.txt	2026-08-03 20:48:51.257905074 +0000
@@ -128,6 +128,7 @@
 find_package(KF6Crash ${KF_MIN_VERSION} REQUIRED)
 find_package(KF6IconThemes ${KF_MIN_VERSION} CONFIG REQUIRED)
 find_package(KF6ColorScheme ${KF_MIN_VERSION} CONFIG REQUIRED)
+find_package(KF6I18n ${KF_MIN_VERSION} CONFIG REQUIRED)
 find_package(KF6DocTools ${KF_MIN_VERSION})
 set_package_properties(
     KF6DocTools
